centreon/www/class/centreonLog.class.php

```
@@ -37,76 +37,44 @@ class CentreonUserLog
-    private $path;
-
     /**
-     * CentreonUserLog constructor
-     *
      * @param int $uid
-     * @param CentreonDB $pearDB
-     *
-     * @throws PDOException
+     * @param CentreonDB $pearDB unused, kept for BC
      */
     public function __construct($uid, $pearDB)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$pearDB'. (undefined)

(UnusedFormalParameter)